tests/page/page-network-request.spec.ts

```diff
@@ -19,6 +19,14 @@ import { test as it, expect } from './pageTest';
 import { attachFrame } from '../config/utils';
 import fs from 'fs';
 
+function adjustServerHeaders(headers: Object, browserName: string, channel: string) {
```

Comment:
```suggestion
function adjustServerHeaders(headers: Object, browserName: string, channel: string) {
```

nit: rewriteServerHeaders?